File: hashing/hash_map_ops.py
my_hash_map = {}

my_hash_map[4] = 8
print(my_hash_map[4]) # Prints 83

print(4 in my_hash_map) # Prints true
print(854 in my_hash_map) # Prints false

my_hash_map[8] = 327
my_hash_map[45] = 82523

for key, val in my_hash_map.items():
    print(f"{key}: {val}")

File: hashing/hash_map_ops.js
(function main() {
    let myHashMap = new Map();

    myHashMap.set(4, 83);
    console.log(myHashMap.get(4)); // Prints 83

    console.log(myHashMap.has(4)); // Prints true
    console.log(myHashMap.has(854)); // Prints false

    myHashMap.set(8, 327);
    myHashMap.set(45, 82523);

    for (const [key, val] of myHashMap)
    {
        console.log(`${key}: ${val}`);
    }
}());